Enable Estates

stackpoint opened the pull request:
Enable estates for more government types

stackpoint committed:
Add vanilla estate files

07_jains.txt
0022_rajput.txt
001_brahmins.txt
01_church.txt
002_maratha.txt
02_nobility.txt
003_vaisyas.txt
03_burghers.txt
04_cossacks.txt
05_nomadic_tribes.txt
06_dhimmi.txt

stackpoint committed:
Enable estates for guilds

Allow non-burgher estates in government types that have merchant republic factions (Aristocrats/Traders/Guilds).

SpiritBaker commented:
Does not work for me. Estate tab is greyed out

SpiritBaker commented:
Nvm it works now, hm 
Only burghers are not enabled.

SpiritBaker commented:
I went through privileges and disabled some of them for trade republics. Like nobles supremacy over the crown, nobles levies etc.  Also added -10% to church and nobility influence.

stackpoint committed:
Enable burghers for guilds

Allow burgher estates in government types that have merchant republic factions (Aristocrats/Traders/Guilds).
